### PR TITLE
New AI Rate Limiting Advanced plugin

### PR DESCRIPTION
**Pull request by AntoineJac**
### Testing instructions

Preview link:
[AI Analytics](https://deploy-preview-7214--kongdocs.netlify.app/gateway/unreleased/production/logging/ai-analytics/)
[Landing page](https://deploy-preview-7214--kongdocs.netlify.app/gateway/unreleased/ai-gateway/)
[AI RLA plugin](https://deploy-preview-7214--kongdocs.netlify.app/hub/kong-inc/ai-rate-limiting-advanced/unreleased/)

### Checklist 

- [x] Review label added 
- [x] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

For example, if this change is for an upcoming 3.6 release, enclose your content in `{% if_version gte:3.6.x %} <content> {% endif_version %}` tags (or `if_plugin_version` tags for plugins). 

Use any of the following keys:
* `gte:<version>` - greater than or equal to a specific version
* `lte:<version>` - less than or equal to a specific version
* `eq:<version>` - exactly equal to a specific version

You can do the same for older versions.

### CODE REVIEW

**Comment by lena-larionova** [comment]
@AntoineJac the build will continue to fail until we have the plugin's schema, which we generate from an RC. Let me get a placeholder file up and you should be able to rebase after to make the build succeed.

File: app/_hub/kong-inc/ai-rate-limiting-advanced/_metadata/_index.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+plus: false
+enterprise: true
+premium: true
+konnect: true
+network_config_opts: Self-managed traditional, DB-less, and hybrid
+notes: |
+  In DB-less, hybrid mode, and Konnect, the <code>cluster</code> config strategy
+  is not supported. Use <code>redis</code> instead.
+categories:
+  - traffic-control
```
Comment (lena-larionova) [inline app/_hub/kong-inc/ai-rate-limiting-advanced/_metadata/_index.yml:19]:
```suggestion
  - traffic-control
  - ai
```

**Review by lena-larionova — CHANGES_REQUESTED** [review]

**Comment by lena-larionova** [comment]
@AntoineJac The AI Gateway general get started guide got split into guide + overview, so I moved your description of the new plugin there. I also added it to the nav and added an icon for the plugin. You can preview the docs here:

[AI Analytics](https://deploy-preview-7214--kongdocs.netlify.app/gateway/unreleased/production/logging/ai-analytics/)
[Landing page](https://deploy-preview-7214--kongdocs.netlify.app/gateway/unreleased/ai-gateway/)
[AI RLA plugin](https://deploy-preview-7214--kongdocs.netlify.app/hub/kong-inc/ai-rate-limiting-advanced/unreleased/)

File: app/_data/tables/version_errors_konnect.yml [hunk 2]
```diff
@@ -2741,3 +2741,11 @@ messages:
...
     DocumentationURL: |
       [Response Rate Limiting plugin](/hub/kong-inc/response-ratelimiting/)
+  - ID: P239
+    Severity: error
+    Description: |
+      Plugin `ai-rate-limiting-advanced` is not available in Kong gateway versions < 3.7.0.0.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.7.0.0` or above.
+    DocumentationURL: |
+      [Ai Rate Limiting Advanced plugin](/hub/kong-inc/ai-rate-limiting-advanced/)
```
Comment (lena-larionova) [inline app/_data/tables/version_errors_konnect.yml:2751]:
```suggestion
```
Let's remove this for now. Konnect updates to this table get generated out of Koko and someone on that team usually updates the whole table right before release.

File: app/_hub/kong-inc/ai-rate-limiting-advanced/overview/_index.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+---
+
+The AI Rate Limiting Advanced plugin provides rate limiting for the providers used by any AI plugins. The
+AI Rate Limiting plugin extends the
+[Rate Limiting Advanced](/hub/kong-inc/rate-limiting-advanced/) plugin.
+
+This token is using the tokens data returned by LLM provider to calculate the queries cost.
+The same HTTP request can vary greatly in cost depending on the calculation of the 
+LLM providers.
+
```
[the remark this answers is not in the document]
Comment (AntoineJac) [reply app/_hub/kong-inc/ai-rate-limiting-advanced/overview/_index.md:12]: It is the tokens return by the llm plugins

File: app/_hub/kong-inc/ai-rate-limiting-advanced/how-to/_index.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+```sh
+curl -i -X POST http://localhost:8001/services/example-service/plugins \
+  --data name=ai-rate-limiting-advanced \
+  --data config.limit=100,10000 \
+  --data config.window_size=60,3600 \
+  --data config.sync_rate=10
+```
+
+The AI Rate Limiting Advanced plugin supports threes rate limiting strategies. The default strategy will estimate cost on queries by counting the total token value returned in the LLM responses.
+
```
[the remark this answers is not in the document]
Comment (AntoineJac) [reply app/_hub/kong-inc/ai-rate-limiting-advanced/how-to/_index.md]: I have just fixed it.

**Comment by AntoineJac** [comment]
@lena-larionova , thanks I reviewed and accepted all your changes. You can add them in your docs. Thanks

**Review by Guaris — APPROVED** [review]

**Review by Guaris — APPROVED** [review]

**Pull request merged**